src/statistiek_hub/resources/observation_resource.py

```diff
@@ -62,16 +62,10 @@ def clean(self, value, row, **kwargs):
         start_date = convert_to_date(row["temporal_date"])
         end_date = add_timedelta(start_date, temporal_type)
 
-        temporal, created = TemporalDimension.objects.get_or_create(
+        temporal = TemporalDimension.objects.get(
```

Comment:
The code now retrieves an existing TemporalDimension from the database, which means we lose the ability to create a new one if it doesn't exist. If this is intentional, what should happen when an item is not found?  Perhaps add a try-except block to log this error when it occurs?

Reply:
the check if a TemporalDimension item exists is already done earlier and raises an exception.